documentation/staging/content/quickstart/create-domain.md

```diff
@@ -2,57 +2,50 @@
 title: "Create a domain"
 date: 2019-02-22T15:44:42-05:00
 draft: false
-weight: 6
+weight: 3
 ---
 
+1. Select a user name and password for the WebLogic domain administrator credentials and use them to create a Kubernetes Secret for the domain:
 
-1. For use in the following steps:
+    ```shell
+    $ kubectl create secret generic sample-domain1-weblogic-credentials \
+      --from-literal=username=YOUR_USERNAME --from-literal=password=YOUR_PASSWORD \
+      -n sample-domain1-ns
+    ```
+
+   Replace `YOUR_USERNAME` and `YOUR_PASSWORD` with your user name and password.
```

Comment:
Just to mitigate the slight chance of confusion with SSO, consider changing "YOUR_" to "ADMIN_" and replacing "your" with "your choice of".

Perhaps add: The password must be at least 8 characters long and must contain at least one non-alphabetical character.